fuel-relayer/src/config.rs

```diff
@@ -0,0 +1,69 @@
+use std::{str::FromStr, time::Duration};
+
+use ethers_core::types::H160;
+
+#[derive(Clone, Debug)]
+pub struct Config {
+    /// number of blocks between eth blocks where deposits/validator set get finalized.
+    /// Finalization is done on eth height measurement.
+    pub eth_finality_slider: u64,
```

Comment:
I think `finality_period` might be a more clear description?

```suggestion
    pub eth_finality_period: u64,
```

Reply:
I like slider better, but lets change it to period as it represent time needed before finalization